### PR TITLE
Adding shared flow note

### PR DESCRIPTION
**Pull request by AlexJerabek**
Office Scripts can't be run through shared Power Automate flows. This PR adds documentation to that effect.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[747eacb](https://github.com/OfficeDev/office-scripts-docs/commits/747eacbc3bd2c20a0334f94f4b32c3e94d6ae63a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PANote/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C25%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202003252255261656-95%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2kjj1XEci8SphvAroKAfH7ZtzOAOaQ0vhpoSLwFXrxk%3D&st=2020-03-25T22%3A51%3A34Z&se=2020-04-25T22%3A56%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/power-automate-integration.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+ms.date: 03/25/2020
 localization_priority: Normal
 ---
 
 # Integrate Office Scripts with Power Automate
 
 [Power Automate](https://flow.microsoft.com) integrates your script into a larger workflow. You can use Power Automate do things like add the contents of an email to a worksheet's table or create actions in your project management tools based on workbook comments. If you are new to Power Automate, we recommend visiting [Get started with Power Automate](https://docs.microsoft.com/power-automate/getting-started). There, you can learn more about automating your workflows across multiple services.
 
+> [!NOTE]
+> Currently, Office Scripts can't be run from a [shared flow](https://docs.microsoft.com/power-automate/share-buttons). Scripts can only be run by the user who created the script, even through Power Automate.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/power-automate-integration.md]: Remember, you can remove docs.microsoft.com and use the relative link.
```suggestion
> Currently, Office Scripts can't be run from a [shared flow](/power-automate/share-buttons). Only the user who created a script can run it, even through Power Automate.
```

Comment (ElizabethSamuel-MSFT) [inline docs/develop/power-automate-integration.md]: Remember, you can remove docs.microsoft.com and use the relative link.
```suggestion
> [!IMPORTANT]
> Currently, you can't run Office Scripts from a [shared flow](/power-automate/share-buttons). Only the user who created a script can run it, even through Power Automate.
```

File: docs/develop/power-automate-integration.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 title: 'Integrate Office Scripts with Power Automate'
 description: 'How to get Office Scripts for Excel on the web working with a Power Automate workflow.'
-ms.date: 01/31/2020
+ms.date: 03/25/2020
 localization_priority: Normal
 ---
 
 # Integrate Office Scripts with Power Automate
 
 [Power Automate](https://flow.microsoft.com) integrates your script into a larger workflow. You can use Power Automate do things like add the contents of an email to a worksheet's table or create actions in your project management tools based on workbook comments. If you are new to Power Automate, we recommend visiting [Get started with Power Automate](https://docs.microsoft.com/power-automate/getting-started). There, you can learn more about automating your workflows across multiple services.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/power-automate-integration.md]:
```suggestion
[Power Automate](https://flow.microsoft.com) integrates your script into a larger workflow. You can use Power Automate do things like add the contents of an email to a worksheet's table or create actions in your project management tools based on workbook comments. If you are new to Power Automate, we recommend visiting [Get started with Power Automate](/power-automate/getting-started). There, you can learn more about automating your workflows across multiple services.
```

**Comment by AlexJerabek** [comment]
Thanks for the reviews @ElizabethSamuel-MSFT. I've included your changes.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[09196b3](https://github.com/OfficeDev/office-scripts-docs/commits/09196b3da000dbda4415edb05daee74bfb2ebfb7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PANote/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C26%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202003261549286184-95%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FIdVX5G1lVe8UeYm5j0bCVnG16rX8qskvh%2FYqx36N%2Bg%3D&st=2020-03-26T15%3A45%3A41Z&se=2020-04-26T15%3A50%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3934cc6](https://github.com/OfficeDev/office-scripts-docs/commits/3934cc630e0a47ca8f48a5f1a88865386ad058d4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PANote/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C26%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202003261549419503-95%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=u2idU3%2FSMkhRD03YaMrs6gN1vdTCqJcYR344WPf%2F%2FNk%3D&st=2020-03-26T15%3A46%3A52Z&se=2020-04-26T15%3A51%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**